nms/api/src/main/java/net/dzikoysk/funnyguilds/nms/api/ProtocolDependentHelper.java

```diff
@@ -0,0 +1,47 @@
+package net.dzikoysk.funnyguilds.nms.api;
+
+import com.viaversion.viaversion.api.Via;
+import org.bukkit.entity.Player;
+
+public class ProtocolDependentHelper {
+    private static final int V1_19_3_PROTOCOL_VERSION = 761;
+
+    private static boolean viaApiAvailable = false;
+
+    static {
+        try {
+            Class.forName("com.viaversion.viaversion.api.ViaAPI");
```

Comment:
Fajnie by było gdybyśmy mogli używać aktualny system do innych hooków niż znowu robić to jakoś oddzielnie